src/ui/reusable/RealmLoadingComponent/index.tsx

```diff
@@ -156,11 +156,12 @@ export abstract class RealmLoadingComponent<
         });
         const credentials = hydrateCredentials(realm.credentials);
         const user = await app.logIn(credentials);
-        return new Realm({
+        return Realm.open({
```

Comment:
Using `Realm.open` here to ensure the schema gets downloaded before the Realm is considered loaded.